spec/views/posts_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Kindly fix your migrattion this, make sure they can run without error. I would suggest you to remove the `add_index: ...` because by default rails add index key on foreigner_keys, that's why you are getting the error bellow

![image](https://github.com/DJ-MrJay/My-Blog-App/assets/65068771/cb266eed-736b-4513-9f45-b7eb74d6ca44)